src/fixes/incorrect_delimiter.rs

```diff
@@ -0,0 +1,102 @@
+use super::Fix;
+use crate::common::*;
+
+pub(crate) struct IncorrectDelimiterFixer<'a> {
+    name: &'a str,
+}
+
+impl Default for IncorrectDelimiterFixer<'_> {
+    fn default() -> Self {
+        Self {
+            name: "IncorrectDelimiter",
+        }
+    }
+}
+
+impl Fix for IncorrectDelimiterFixer<'_> {
+    fn name(&self) -> &str {
+        self.name
+    }
+
+    fn fix_line(&self, line: &mut LineEntry) -> Option<()> {
+        let cleaned_key = remove_invalid_leading_chars(&line.get_key()?);
+
+        let fixed_key: String = cleaned_key
```

Comment:
In this case `IncorrectDelimiterFixer` will fix also `LeadingCharacter` warnings but it's task of `LeadingCharacterFixer`.
What if the user specifies to ignore the `LeadingCharacter` check, but gets the fixes, :thinking: 

We should leave leading chars as it is in my opinion.  
Maybe add a test for such cases?

Reply:
That's a good point, I can skip those chars and then apply the delimiter fixer afterwards.

Reply:
I think I've come up with a solution, please take a look 🙏 . I added a test case to ensure correct behavior.

It won't handle invalid suffixes, but that should be left to another checker/fixer?